lib/views/pages/splash_screen.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+import 'package:sun_flutter_capstone/views/pages/dashboard.dart';
+import 'package:sun_flutter_capstone/views/pages/session/extra_logo.dart';
+import 'package:sun_flutter_capstone/views/pages/session/register.dart';
+
+class SplashScreen extends StatefulHookConsumerWidget {
+  final bool authenticated;
+  const SplashScreen({Key? key, required this.authenticated}) : super(key: key);
+
+  @override
+  ConsumerState<ConsumerStatefulWidget> createState() => _SplashScreenState();
+  // _SplashScreenState createState() => _SplashScreenState();
+}
+
+class _SplashScreenState extends ConsumerState<SplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(
+      Duration(seconds: 3),
+      () => Navigator.push(
+        context,
+        MaterialPageRoute(builder: (context) => widget.authenticated ? Dashboard() : Register()),
```

Comment:
```suggestion
        MaterialPageRoute(
            builder: (context) => widget.authenticated ? Dashboard() : Register(),
        ),
```

add comma to trigger formatting